openapi/azure.openai.chat/.gitignore

```diff
@@ -0,0 +1,3 @@
+target
+generated
+Config.toml
```

Comment:
Shall we remove these

Reply:
Addressed in f713c6745a129c25a6eeda2a853911c8e60245c8